modular_skyrat/modules/colony_fabricator/code/design_datums/flatpack_machines.dm

```diff
@@ -155,8 +155,10 @@
 	id = "flatpack_rtg"
 	build_type = COLONY_FABRICATOR
 	materials = list(
-		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 5,
-		/datum/material/uranium = SHEET_MATERIAL_AMOUNT * 2
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 15,
+		/datum/material/uranium = SHEET_MATERIAL_AMOUNT * 5,
+		/datum/material/plasma = HALF_SHEET_MATERIAL_AMOUNT,
+		/datum/material/gold = HALF_SHEET_MATERIAL_AMOUNT,
```

Comment:
This feels much more reasonable, and will make it less the go-to option when you need more than just a crumb of power.